### PR TITLE
Feature/update magma

### PR DESCRIPTION
**Pull request by MurashovVen**
- Magma is updated. Now [this](https://github.com/magma/augmented-networks) is used.
- Simplified Consumer.HSS verification procedure.
- Provider.AccessPoint service is silenced.

### CODE REVIEW

File: code/consumer/registration/const.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package registration
+
+import (
+	"github.com/0chain/gosdk/core/logger"
+)
+
+// intLevelFromStr converts string log level to gosdk logger level int value.
+func intLevelFromStr(level string) int {
```
Comment (platsko) [inline code/consumer/registration/const.go:8]: I think name `logLevelFromStr` is better )

File: code/magma/server/grpc/magma.go [hunk 2]
```diff
@@ -146,20 +137,20 @@ func notifyNewProvider(ctx context.Context, req *magma.ConnectRequest, proxyAddr
...
-func newSessionBilling(ctx context.Context, req *magma.ConnectRequest, acknowledgmentID, sessionID, proxyAddr string) error {
+func newSessionBilling(ctx context.Context, req *magma.Session, idMaps config.IDMaps, acknowledgmentID, proxyAddr string) error {
 	cl, err := pproxy.Client(proxyAddr)
 	if err != nil {
 		msg := "can not connect to provider Proxy while making NewSessionBilling request"
 		return status.Error(codes.Internal, msg)
 	}
 
-	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
+	ctx, cancel := context.WithTimeout(ctx, 20*time.Second)
```
Comment (platsko) [inline code/magma/server/grpc/magma.go:147]: Timeout value - could it be moved to the config yaml file?

File: code/magma/server/grpc/magma.go [hunk 3]
```diff
@@ -93,41 +89,36 @@ func verifyUser(ctx context.Context, req *magma.ConnectRequest, hssAddr string)
...
-func notifyNewProvider(ctx context.Context, req *magma.ConnectRequest, proxyAddr, sessionID string) (string, error) {
+func notifyNewProvider(ctx context.Context, req *magma.Session, idMaps config.IDMaps, proxyAddr string) (string, error) {
 	cl, err := cproxy.Client(proxyAddr)
 	if err != nil {
 		msg := "can not connect to consumer Proxy while making NotifyNewProvider request"
 		return "", status.Error(codes.Internal, msg)
 	}
 
-	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
+	ctx, cancel := context.WithTimeout(ctx, 20*time.Second)
```
Comment (platsko) [inline code/magma/server/grpc/magma.go:115]: Timeout value - could it be moved to the config yaml file?

File: code/provider/magma/client.go [hunk 4]
```diff
@@ -1,18 +1,14 @@
...
+//	conn, err := grpc.Dial(addr, grpc.WithInsecure())
+//	if err != nil {
+//		return nil, err
+//	}
+//
+//	client := magma.NewMagmaClient(conn)
+//
+//	return client, nil
+//}
+// TODO remove or update accordance with magma
```
Comment (platsko) [inline code/provider/magma/client.go:14]: I think you can remove this file.

File: code/provider/registration/const.go [hunk 5]
```diff
@@ -0,0 +1,24 @@
+package registration
+
+import (
+	"github.com/0chain/gosdk/core/logger"
+)
+
+// intLevelFromStr converts string log level to gosdk logger level int value.
+func intLevelFromStr(level string) int {
```
Comment (platsko) [inline code/provider/registration/const.go:8]: `logLevelFromStr`

File: code/provider/server/grpc/ap.go [hunk 6]
```diff
@@ -37,33 +30,36 @@ func newAccessPointServer(cfg *config.Config) *accessPointServer {
...
-	if resp.Status == magma.ConnectionStatus_Failed {
-		return nil, status.Errorf(codes.Unknown, "requesting Magma with Connect request ended with \"Failed\" status")
-	}
-
-	log.Logger.Info("Access Point: Handling Connect successfully ended.")
-
-	return &provider.BeginSession{SessionID: resp.SessionID, Status: resp.Status}, nil
+	// TODO remove or update accordance with magma.
+	return nil, errors.New("unimplemented")
 }
```
Comment (platsko) [inline code/provider/server/grpc/ap.go:65]: I think you can remove this file.

**Review by platsko — CHANGES_REQUESTED** [review]

File: code/consumer/registration/const.go (hunk 1 above)
Comment (MurashovVen) [reply code/consumer/registration/const.go:8]: Renamed )

File: code/magma/server/grpc/magma.go (hunk 2 above)
Comment (MurashovVen) [reply code/magma/server/grpc/magma.go:147]: Sure. I've moved timeouts to config.

File: code/magma/server/grpc/magma.go (hunk 3 above)
Comment (MurashovVen) [reply code/magma/server/grpc/magma.go:115]: Sure. I've moved timeouts to config.

File: code/provider/magma/client.go (hunk 4 above)
Comment (MurashovVen) [reply code/provider/magma/client.go:14]: Removed.

File: code/provider/registration/const.go (hunk 5 above)
Comment (MurashovVen) [reply code/provider/registration/const.go:8]: Renamed.

File: code/provider/server/grpc/ap.go (hunk 6 above)
Comment (MurashovVen) [reply code/provider/server/grpc/ap.go:65]: Removed.

**Review by platsko — APPROVED** [review]

**Pull request merged**